cves/CVE-2012-4387.yml

```diff
@@ -60,7 +65,7 @@ fixes_vcc_instructions: |
   Please put the SVN commit number in "commit" below, and any notes about how this
   was discovered in the "note" field.
 fixes:
-  - commit: 87935af56a27235e9399308ee1fcfb74f8edcefa
+  - commit: 80e03182d66d9e6ab18f9a9a9b3c42725a1c89e9
     note: Restricted parameter length
```

Comment:
Does this note still apply after changing the fix commit?

Reply:
It does indeed! The fix itself was restricting the parameter length, but the commit hash for this fix was not originally pointing to the correct commit.